actioncable/test/client_test.rb

```diff
@@ -99,6 +106,35 @@ def with_puma_server(rack_app = ActionCable.server, port = 3099)
     end
   end
 
+  if defined?(::Iodine)
+    def with_iodine_server(rack_app = ActionCable.server, port = 3099, block)
+      begin
+        ::Iodine.listen2http(app: rack_app, port: port.to_s, address: "127.0.0.1")
+        ::Iodine.workers = 1 # don't cluster the test
+        ::Iodine.threads = 1 # one for the server another for the task
+        t = Thread.new { ::Iodine.start }
+        block.call(port)
+      rescue
+        nil
```

Comment:
This rescue swallows all exceptions. Omit this clause entirely.

Reply:
Right, sorry 👍🏻